app/assets/stylesheets/layouts/_countries.scss

```diff
@@ -217,7 +217,7 @@
 .tab-countries {
   background-color: #f1f1f1;
   ul {
-    max-width: 83.33333%;
+    max-width: 1080px;
```

Comment:
Is this the max layout size?

I guess it is still there because some legacy code becuase the .row should be the one that fits that size... anyway could we get, at least, it from settings?